webapp/client/src/pages/RegistrationPage.js

```diff
@@ -188,7 +197,7 @@ RegistrationPage.propTypes = {
   stepHeader: PropTypes.exact({
     // TODO: define these here, not in Python
     title: PropTypes.string,
-    intro: PropTypes.string,
+    intro: PropTypes.element,
```

Comment:
I do not think this should have to change. Defined here are the types of the *input* to the RegistrationPage component. In fact we don't even need an intro here anymore - as the text is set in the component itself, instead of in Flask.

Reply:
Fixed that, deleted the prop type :)